modules/trusted-firmware-m/Kconfig.psa.defconfig

```diff
@@ -115,10 +115,10 @@ config PSA_WANT_ECC_BRAINPOOL_P_R1_256
 	default y
 config PSA_WANT_ECC_BRAINPOOL_P_R1_384
 	bool
-	default y
```

Comment:
You can remove the default line completely, right? Why do you need to keep it?

Reply:
Technically, yes. But this file sets the default configurations to either y or n, so having it here for completeness, also in case they original default changes this would not be affected by that.